blog/2021-08-26-welcome/index.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: welcome
+title: Welcome
+authors: [gableroux, davidmfinol]
+tags: [facebook, hello, docusaurus]
+---
+
+[Docusaurus blogging features](https://docusaurus.io/docs/blog) are powered by the [blog plugin](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog).
+
+Simply add Markdown files (or folders) to the `blog` directory.
+
+Regular blog authors can be added to `authors.yml`.
+
+The blog post date can be extracted from filenames, such as:
+
+- `2019-05-30-welcome.md`
+- `2019-05-30-welcome/index.md`
+
+A blog post folder can be convenient to co-locate blog post images:
+
+![Docusaurus Plushie](./docusaurus-plushie-banner.jpeg)
```

Comment:
While I like being able to add images we should probably make sure that we commit them to LFS from the start (and also squash this PR), to not slow down the repo by unnecessary deltas.

Suggest fixing that as part of this PR